static/js/unread_ops.js

```diff
@@ -36,6 +41,13 @@ export function mark_all_as_read() {
                 save_compose: true,
             });
```

Comment:
Does the `reload.initiate` replace the banner with a "Reloading..."` notice?

Reply:
Yes, they all use the same DOM element, so no change of overlapping or multiple notices.